content/parts/en/datasets.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "Datasets"
+html_description: "Download datasets"
```

Comment:
Reuse the header link title and description 🙂 Ideally through reusing the locales data from `header.json`, so we don't duplicate data and ease further translations

Reply:
> Reuse the header link title and description 🙂 Ideally through reusing the locales data from `header.json`, so we don't duplicate data and ease further translations

Following our synch point I have updated the html description :+1: